### ISSUE TITLE
dplyr revdep checks: Variable context not set

### ISSUE THREAD

**Issue by lionel-** [lwjohnst86/carpenter#6]
Hello,

I see in dplyr's revdep checks for the 0.6 release:

```r
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  17: freduce(value, `_function_list`) at /tmp/RtmpT6Czo8/R.INSTALL13c163d710cb/magrittr/R/pipe.R:28
  18: function_list[[i]](value) at /tmp/RtmpT6Czo8/R.INSTALL13c163d710cb/magrittr/R/freduce.R:17
  19: dplyr::mutate_each_(., dplyr::funs(as.character), dplyr::everything())
  20: is_empty(vars) at /home/muelleki/git/R/dplyr/R/colwise-mutate.R:251
  21: dplyr::everything() at /tmp/RtmpK4tYqb/devtools25fa524fad49/tidyverse-rlang-684221a/R/types.R:240
  22: current_vars() at /home/muelleki/git/R/dplyr/R/select-utils.R:129
  23: cur_vars_env$selected %||% abort("Variable context not set") at /home/muelleki/git/R/dplyr/R/select-utils.R:46
  24: abort("Variable context not set") at /tmp/RtmpK4tYqb/devtools25fa524fad49/tidyverse-rlang-684221a/R/operators.R:14
```

This is most likely caused by using `everything()` within an underscored SE variant like `select_()` or `mutate_each_()`.  An easy fix is to quote the call like this: `~everything()`.

**Comment by lwjohnst86** [comment]
Thanks for that! I've been travelling so haven't been able to update the package in light of the new dplyr version, but I'll get to this quickly end of this month!

**Comment by lwjohnst86** [comment]
Fixed #7
